pom.xml

```diff
@@ -118,6 +118,12 @@
             <version>3.23.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.instancio</groupId>
+            <artifactId>instancio-junit</artifactId>
```

Comment:
Do we need this?

Reply:
Yes, it is used heavily in the tests

Reply:
Oh GitHub didn't show me the test file you created - ignore this comment